nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss

```diff
@@ -19,11 +19,15 @@
 // For more information: https://m2.material.io/design/color/the-color-system.html
 @use '@angular/material' as mat;
 
+// Define some variables that are re-used throughout the theme.
+$on-surface-dark: rgba(black, 0.87);
+$on-surface-light: #ffffff;
+
 // The $material-primary-light-palette define the PRIMARY AND ACCENT palettes for all Angular Material components used throughout Apache NiFi
 $material-primary-light-palette: (
```

Comment:
These same changes need to be made in the purple.scss theme file. As it stands right now the purple theme light and dark mode look like:

<img width="1728" alt="Screenshot 2024-03-08 at 8 44 12 PM" src="https://github.com/apache/nifi/assets/6797571/1d12cfed-3651-4baf-b1ab-e427153c1cb1">
<img width="1728" alt="Screenshot 2024-03-08 at 8 44 26 PM" src="https://github.com/apache/nifi/assets/6797571/e6d5b887-45b2-4956-9786-ff4c1344e971">

Reply:
Looks good now.